src/utils/hours.js

```diff
@@ -90,12 +90,14 @@ export function findHoursSetByNodeForNow({ node, now }) {
       return null;
     }
 
-    // Star and end of hour set range.
-    const start = set.field_date_range.value;
-    const end = set.field_date_range.end_value;
+    // Start and end of hour set range.
+    const start = parseISO(set.field_date_range.value);
+    const end = parseISO(set.field_date_range.end_value);
 
     // Check if "now" is within the range including start and end days.
-    return now.isSameOrAfter(start, 'day') && now.isSameOrBefore(end, 'day');
+    return (
+      isSameDay(now, start) || isSameDay(now, end) || (now > start && now < end)
```

Comment:
```suggestion
      !(now < start || now > end)
```
I believe this line could be simplified to check if now is neither before the start or after the end.